llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1598,6 +1628,29 @@ void VPReductionRecipe::print(raw_ostream &O, const Twine &Indent,
     O << " (with final reduction value stored in invariant address sank "
          "outside of loop)";
 }
+
+void VPReductionEVLRecipe::print(raw_ostream &O, const Twine &Indent,
```

Comment:
can you also add vplan test for that recipe ?

Reply:
1de39ab970a3d0094da01a08e68208b77d95224f
For now, I repeated the test of VPReductionRecipe. If there is anything else VPReductionEVLRecipe need to be tested, please let me know.